src/components/NewBookForm.js

```diff
@@ -47,31 +43,23 @@ const AddForm = () => {
   };
 
   return (
-    <form onSubmit={handleAdd}>
-      <label htmlFor="bookTitle">
-        Title
-        <input type="text" id="bookTitle" placeholder="Title" value={title} onChange={handleTitleChange} />
-      </label>
-      <br />
-      <label htmlFor="bookAuthor">
-        Author
-        <input type="text" id="bookAuthor" placeholder="Author" value={author} onChange={handleAuthorChange} />
-      </label>
-      <br />
-      <label htmlFor="bookCategory">
-        Category
-        <select id="bookCategory" value={category} onChange={handleCategoryChange}>
-          {categories.map((category) => (
-            <option key={category} value={category}>
-              {category}
-            </option>
-          ))}
-        </select>
-      </label>
-      <br />
-      <button type="submit">Add Book</button>
-    </form>
+    <div className="form-container">
+      <h2 className="form-title">
+        ADD A NEW BOOK
+      </h2>
+      <form className="add-form" onSubmit={handleAdd}>
+        <label className="grow-3" htmlFor="bookTitle">
+          <input className="input-control input-title" type="text" id="bookTitle" placeholder="Title" value={title} onChange={handleTitleChange} required />
+        </label>
+        <br />
+        <label className="grow-3" htmlFor="bookAuthor">
+          <input className="input-control input-author" type="text" id="bookAuthor" placeholder="Author" value={author} onChange={handleAuthorChange} required />
+        </label>
```

Comment:
### As you can see in the screenshot, there's a category field instead of an author field and it's not a normal field, it's a select element used for that. Kindly change that 👍 

<img width="192" alt="Screenshot 2023-06-17 at 11 42 40 PM" src="https://github.com/MarcoDDM/booksStore/assets/83048208/1e5d6a99-8ade-4630-8c92-9f1c1921665b">

### [CONTEXT]
<img width="747" alt="Screenshot 2023-06-17 at 11 44 28 PM" src="https://github.com/MarcoDDM/booksStore/assets/83048208/15dd24cb-b72b-4372-9a05-68733b0a2c54">

Reply:
Ohh thanks for the hint